pentaho5/src/main/java/pt/webdetails/cpk/InterPluginBroker.java

```diff
@@ -33,7 +35,7 @@
   private static final String CDE_RENDER_API_RENDER_METHOD_TAG = "cde-render-api-render-method";
   private static final String CDE_RENDER_API_RENDER_METHOD = "render";
 
-  private static IPluginCall cdeRenderApiCall;
+  private static IPluginCall pluginCall;
```

Comment:
I liked the first name better, `cdeRenderApiCall` or `cdeRenderCall`. Its more explicti